### PR TITLE
Tests for parsing results

### PR DESCRIPTION
**Pull request by tiginamaria**
Add test for parsing hyperstyle and qodana reports

### CODE REVIEW

File: analysis/test/python/evaluation/qodana/test_parse_result.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+]
+
+
+@pytest.mark.parametrize(('result_file', 'solutions_count', 'issues_count'), QODANA_OUTPUT_DATA)
+def test_parse_qodana_result(result_file: Path, solutions_count: int, issues_count: List[int]):
+    df_result = parse_qodana_result(RESOURCES_PATH / result_file)
+    assert df_result.shape[0] == solutions_count
+    for i, result in enumerate(df_result.iterrows()):
+        report = QodanaReport.from_str(result[1][SubmissionColumns.QODANA_ISSUES.value])
+        assert len(report.list_problem) == issues_count[i]
```
[the remark this answers is not in the document]
Comment (nbirillo) [reply analysis/test/python/evaluation/qodana/test_parse_result.py:25]: What do you think?

**Review by nbirillo — CHANGES_REQUESTED** [review]
Great!

File: analysis/test/python/evaluation/qodana/test_parse_result.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (tiginamaria) [reply analysis/test/python/evaluation/qodana/test_parse_result.py]: Add special methods for common tests running and config generation.

**Review by nbirillo — APPROVED** [review]
Great, thank you!

**Pull request merged**